<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="temprature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Gestion de stock</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>781</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">tabWidget_2 = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajout</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>date_stk</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantite">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prix">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dat">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tab_stk_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>471</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_recherche">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>50</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_stock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date__stk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>quantite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prix</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_recherche">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_supprimer">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>140</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>id_stock</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>400</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>date_stk</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>460</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>quatite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>370</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Stock</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prix_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>430</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>430</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>prix</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>430</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="colone_tri">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>90</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_stock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>produit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>quantite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>livreur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>total</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date_arrivee</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="ordre_tri">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>90</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>ASC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DESC</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="button_tri">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>tri</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="dat_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>400</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantite_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>460</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des fournisseurs</string>
     </attribute>
     <widget class="QTabWidget" name="affichage2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Ajout</string>
       </attribute>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="tel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>80</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="adresse">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>170</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tab_fourn_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>441</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_recherche_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_f</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>adresse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom_soc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>numero</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_recherche_2">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QComboBox" name="colone_tri_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>70</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_f</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>adresse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom_soc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>numero</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="ordre_tri_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>70</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>ASC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DESC</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="button_tri_2">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>tri</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_supprimer_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>130</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>id_f</string>
        </property>
       </widget>
       <widget class="QPushButton" name="imprimer_2">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>statistique</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_39">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>300</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_f</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>330</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_41">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_42">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>390</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>tel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numventemodif">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantmodif">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>360</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="idmodif">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>390</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prod_modif">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>330</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>430</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>430</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
